src/components/item/IFXItemListMixin.js

```diff
@@ -93,13 +94,17 @@ export default {
       }
       return items
     },
-    // TODO: this is inefficient because it's checking all attributes
-    // Make it check only relevant fields
-    // Taken almost directly from the Vuetify docs
     filterSearch(v, s) {
       let search = s
+      if (v && typeof v === 'object' && !Array.isArray(v) && v.data && this.deepSearch) {
```

Comment:
It's totally nit-picky but I'd put `this.deepSearch` first in the `if` statement since that's the thing that is controlling that code.

Reply:
Fixed